contrib/pkg/utils/ovirt/ovirt.go

```diff
@@ -0,0 +1,32 @@
+package ovirt
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"k8s.io/client-go/util/homedir"
+
+	"github.com/openshift/hive/pkg/constants"
+)
+
+// GetCreds reads oVirt credentials either from the specified credentials file,
+// or ~/.ovirt/ovirt-config.yaml
+func GetCreds(credsFile string) ([]byte, error) {
+	if credsFile == "" {
+		for _, filePath := range []string{filepath.Join(homedir.HomeDir(), ".ovirt", constants.OvirtCredentialsName)} {
+
+			_, err := os.Stat(filePath)
+			if err != nil && !os.IsNotExist(err) {
+				return nil, err
+			}
+			if os.IsNotExist(err) {
+				continue
+			}
+			credsFile = filePath
+			break
+		}
+	}
+	// log.WithField("credsFile", credsFile).Info("Loading oVirt creds")
```

Comment:
Did you mean to leave this comment in?

Reply:
I'll remove this. There is a similar log like this for another platform but it was messing up `-o yaml`.